tests/integration/targets/ios_user/tests/cli/auth.yaml

```diff
@@ -12,14 +12,22 @@
     - name: reset connection with {{ ansible_user }}
       meta: reset_connection
 
+    # - name: test login for {{ ansible_user }} user with password
+    #   cisco.ios.ios_command:
+    #     commands:
+    #       - show version
+    #   vars:
+    #     ansible_user: auth_user
+    #     ansible_ssh_pass: pass123
+    #     ansible_private_key_file: ""
```

Comment:
is the plan to fix this, if not lets remove

Reply:
I add the same test case in a different way, made it same as nxos